code/modules/surgery/organs/augments_internal.dm

```diff
@@ -8,17 +8,17 @@
 	var/implant_overlay
 	tough = 1 //not easyly broken by combat damage
 	sterile = 1 //not very germy
+	robotic = 2 // these are cybernetic after all
```

Comment:
Robotic decalares the organ well..robotic..rather then organic... i think theres a few levels 1 simple being kinda 'assisted', 2 being full on robotic. We could make it a define but that might be byond PR scope.